positron/comms/ui-frontend-openrpc.json

```diff
@@ -216,6 +216,26 @@
 				}
 			]
 		},
+		{
+			"name": "execute_command_await",
```

Comment:
I think we should avoid having two different versions of `execute_command` (one that waits and one that doesn't) -- instead we should just have one that always waits. The client side of the UI comm (i.e. R/Python) can decide whether to wait for the RPC to return or not.